en/file_formats/mission_plan.md

```diff
@@ -0,0 +1,168 @@
+# Mission/Flight-Plan File Format (Mission Protocol)
+
+Mission/Flight-Plan files are formatted using JSON, and contain both the mission and associated (optional) geo-fence and rally-point information.
+
+<!-- > doc originates from https://dev.qgroundcontrol.com/en/file_formats/plan.html -->
+
+## Plan (Top Level) {#plan}
+
+The top level structure of the plan file is shown below. 
+
+```
+{
+    "fileType": "Plan",
+    "version": 1
+    "groundStation": "QGroundControl",
+    "mission": {
+        "version": 2
+        "firmwareType": 12,
+        "vehicleType": 2,
+        "cruiseSpeed": 15,
+        "hoverSpeed": 5,
+        "plannedHomePosition": [
+            47.632939716176864,
+            -122.08905141,
+            40
+        ],
+        "items": [
+            ...
+        ],
+    },
+    "geoFence": {
+        ...
+    },
+    "rallyPoints": {
+        ...
+    },
+}
+```
+
+The main fields are:
+
+Key | Description
+--- | ---
+fileType | Must be `Plan`.
+version | The file-format version. Current version is 1.
+groundStation | The name of the ground station that created this file.
+[mission](#mission) | The mission associated with this flight plan.
+[geoFence](#geoFence) (Optional)| The geofence information associated with this flight plan.
+[rallyPoints](#rallyPoints) (Optional) | The rally (safe) points associated with this flight plan.
+
+
+## Mission Object {#mission}
+
+The following values are required:
+
+Key | Description
+--- | ---
+version | The version of the mission object. Current version is 2.
+firmwareType | The firmware type for which this mission was created. This is one of the [MAV_AUTOPILOT](../messages/common.md#MAV_AUTOPILOT) enum values. 
+vehicleType | The vehicle type for which this mission was created. This is one of the [MAV_TYPE](../messages/common.md#MAV_TYPE) enum values.
+cruiseSpeed | The default cruise speed for the mission (metres/second).
+hoverSpeed | The default hover speed for the mission (metres/second). <!-- what is hover speed? Only for Planes? What if on a still-hover system like MC) -->
+plannedHomePosition | The planned home position to show on the map when you are editing the mission. Values with array are latitude, longitude and altitude.
+items | The list of mission item objects associated with the mission. Each item will either be a [SimpleItem](#SimpleItem) or [ComplexItem](#ComplexItem). <!-- can you mix them? -->
```

Comment:
Can the list of items mix complex and simple items in the one plan?

Reply:
Cruise speed is fixed wing or VTOL forward flight. Hover speed is multi-rotor speed.

> The list of mission item objects associated with the mission. Each item will either be a [SimpleItem](#SimpleItem) or [ComplexItem](#ComplexItem).

> Can the list of items mix complex and simple items in the one plan?

Yes.